### DISCUSSION TITLE
Website works locally but GitHub Pages show an empty page

### DISCUSSION THREAD

**Discussion by Jurek33** [community/community#21797] [Other Feature Feedback, Questions, & Ideas]
<p>Hello everybody, I have an issue which I was trying to solve for last few days. Here is my simple website repo: <a href="https://github.com/Jurek33/Robo_Friends" rel="nofollow noopener">https://github.com/Jurek33/Robo_Friends</a> . Here is the whole story: </p>
<ol>
<li>
<p>I created a website locally using create react app. </p>
</li>
<li>
<p>I put it into my new github repo and followed the documentation to publish it using github pages.</p>
</li>
<li>
<p>It used to work fine before I decided to make some changes in a few files and push them to the repo.</p>
</li>
<li>
<p>After that I have noticed that changes do not apply on github pages and I still see the same website as before.</p>
</li>
<li>
<p>I figured out that I have 2 branches (master and gh-pages). Then I realized that I have beed applying all the changes to master branch but my website was publishing from gh-pages branch source. So I switch the source to master branch and deleted gh-pages branch</p>
</li>
<li>
<p>Since then it does not uplad an html file and I only see an empty page. Here are an errors I see on a console  </p>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/45589506b561e92b7e14cd4aaf4c5f0e3dafe41e.png" data-download-href="/uploads/short-url/9TsJn45Dtlxp2tJCopJAZMHnWNw.png?dl=1" title="Screen Shot 2020-03-16 at 1.54.02 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52718116/181066928-9f8bb061-1e96-40d9-9d94-203600e6acc9.png" alt="Screen Shot 2020-03-16 at 1.54.02 PM.png" data-base62-sha1="9TsJn45Dtlxp2tJCopJAZMHnWNw" width="690" height="349" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/45589506b561e92b7e14cd4aaf4c5f0e3dafe41e_2_690x349.png, https://aws1.discourse-cdn.com/github/optimized/1X/45589506b561e92b7e14cd4aaf4c5f0e3dafe41e_2_1035x523.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/45589506b561e92b7e14cd4aaf4c5f0e3dafe41e_2_1380x698.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/45589506b561e92b7e14cd4aaf4c5f0e3dafe41e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-03-16 at 1.54.02 PM.png</span><span class="informations">2880×1458 209 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>Here are the soultions I already tried but they did not work for me:
</li>
<li>
<p>Choosing a Jekyll theme (on some forums people say that it could be a problem even if the website does not use Jekyll)</p>
</li>
<li>
<p>Putting an html file into main folder</p>
</li>
<li>
<p>Changing github pages source (no matter what source I choose it is still not working)</p>
</li>
<li>
<p>Changing an html file path </p>
</li>
<li>
<p>Deleting an old repository and putting all local files into a new repository</p>
</li>
<li>
<p>Cleaning chrome browser cache</p>
</li>
</ol>
<p>If someone knows what am I doing wrong and why I get this errors please let me know what is it. I will appreciate any help.</p>

**Comment by Jurek33** [comment, marked as answer]
<p>Problem was solved when I found this resource (<a href="https://create-react-app.dev/docs/deployment/#github-pages-https-pagesgithubcom" rel="nofollow noopener">https://create-react-app.dev/docs/deployment/#github-pages-https-pagesgithubcom</a> ) and figured out that I need to install gh-pages package which creates gh-pages branch in my github repo and follow the instructions.</p>

**Comment by bilouStrike** [comment]
Facing the some issue: 
My Github repo: https://github.com/bilouStrike/sudoku-app
Github pages for my app : https://biloustrike.github.io/sudoku-app/
